apps/OboeTester/app/src/main/java/com/google/sample/oboe/manualtest/StreamConfiguration.java

```diff
@@ -146,26 +146,27 @@ public void setPerformanceMode(int performanceMode) {
         this.mPerformanceMode = performanceMode;
     }
 
-    public int getInputPreset() {
-        return mInputPreset;
-    }
-    public void setInputPreset(int inputPreset) {
-        this.mInputPreset = inputPreset;
-    }
-
     static String convertPerformanceModeToText(int performanceMode) {
```

Comment:
Couldn't you use `oboe::convertToText` for this?

Reply:
It would be more work to call through JNI than to just add the Java code. Also I may want shorter text.